File: onboardingscreenrecyclerviewjava/src/main/java/com/example/onboardingscreenrecyclerviewjava/GetStarted.java
package com.example.onboardingscreenrecyclerviewjava;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import com.example.onboardingscreenrecyclerviewjava.databinding.ActivityGetStartedBinding;

public class GetStarted extends AppCompatActivity {
    private Button startButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityGetStartedBinding binding = ActivityGetStartedBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        startButton = binding.startButton;
        startButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(GetStarted.this, MainActivity.class);
                startActivity(i);
                finish();
            }
        });
    }
}

File: onboardingscreenrecyclerviewkotlin/src/main/java/com/example/onboardingscreenrecyclerviewkotlin/GetStarted.kt
package com.example.onboardingscreenrecyclerviewkotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import com.example.onboardingscreenrecyclerviewkotlin.databinding.ActivityGetStartedBinding

class GetStarted : AppCompatActivity() {
    lateinit var startButton: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityGetStartedBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.startButton.setOnClickListener(View.OnClickListener {
            val i = Intent(this@GetStarted, com.example.onboardingscreenrecyclerviewkotlin.MainActivity::class.java)
            startActivity(i)
            finish()
        })
    }
}